packages/embed/webpack.dev.js

```diff
@@ -51,7 +51,7 @@ module.exports = {
       title: 'Gr4vy - Embed',
     }),
     new DefinePlugin({
-      PACKAGE_VERSION: 'dev',
+      PACKAGE_VERSION: JSON.stringify('dev'),
```

Comment:
@douglaseggleton what's this for?

Reply:
We introduced package version but it doesn't work with `dev` as its not actually defined, it should be a string, e.g. `"dev"`

Reply:
Isn't `'dev'` already a string?

Reply:
`PACKAGE_VERSION: '"dev"'`

Reply:
So confused. Don't let this top merging though.

Reply:
Take a look at https://webpack.js.org/plugins/define-plugin/#usage - Will probably explain it better than me! :D